recipes/perl-text-csv/meta.yaml

```diff
@@ -0,0 +1,27 @@
+package:
+  name: perl-text-csv
+  version: "1.33"
+
+source:
+  fn: Text-CSV-1.33.tar.gz
+  url: http://search.cpan.org/CPAN/authors/id/M/MA/MAKAMAKA/Text-CSV-1.33.tar.gz
```

Comment:
No problem - I'm discovering recipes and missing important points ! ;-)
Done - 701afdf